Resfeber/Model Controller/ProfileController.swift

```diff
@@ -311,35 +308,36 @@ class ProfileController {
             }
             return
         }
-        
-        let dataTask = URLSession.shared.dataTask(with: request) { (data, response, error) in
-            
+
+        let dataTask = URLSession.shared.dataTask(with: request) { _, response, error in
+
             defer {
                 DispatchQueue.main.async {
                     completion()
                 }
             }
-            
+
             if let error = error {
                 NSLog("Error adding profile: \(error)")
             }
-            
+
             if let response = response as? HTTPURLResponse,
-                response.statusCode != 200 {
+               response.statusCode != 200
+            {
```

Comment:
I think the opening brace should remain on the preceding line to keep consistent with how the rest of the code is formatted.